pkg/debugger/policy_test.go

```diff
@@ -1,71 +1,38 @@
 package debugger
 
 import (
-	"fmt"
 	"net/http/httptest"
 	"testing"
-	"time"
 
+	gomock "github.com/golang/mock/gomock"
```

Comment:
we don't need an import alias here.

Reply:
fixing

Reply:
Don't worry about it, can be handled later. We have other occurrences of this.